### PR TITLE
*: Move statement tests to executor package

### PR DESCRIPTION
**Pull request by zimulala**

### CODE REVIEW

File: parser/parser_test.go [hunk 1]
```diff
@@ -367,11 +367,6 @@ func (s *testParserSuite) TestBuiltin(c *C) {
 		{`SELECT LOCATE('bar', 'foobarbar');`, true},
 		{`SELECT LOCATE('bar', 'foobarbar', 5);`, true},
 
-		// For delete statement
-		{"DELETE t1, t2 FROM t1 INNER JOIN t2 INNER JOIN t3 WHERE t1.id=t2.id AND t2.id=t3.id;", true},
```
Comment (shenli) [inline parser/parser_test.go]: Why remove those cases?

Comment (zimulala) [inline parser/parser_test.go]: remove duplicate tests

**Comment by shenli** [comment]
LGTM

**Comment by coocood** [comment]
LGTM

**Pull request merged**